big_tests/tests/mam_SUITE.erl

```diff
@@ -244,15 +242,15 @@ configurations() ->
 %% Called by test-runner for autocompletion
 all_configurations() ->
     cassandra_configs(true)
-    ++ rdbms_configs(true)
+    ++ rdbms_configs(true, true)
     ++ elasticsearch_configs(true).
 
 configurations_for_running_ct() ->
     cassandra_configs(is_cassandra_enabled(host_type()))
-    ++ rdbms_configs(mongoose_helper:is_rdbms_enabled(host_type()))
+    ++ rdbms_configs(mongoose_helper:is_rdbms_enabled(host_type()), mongoose_helper:is_mnesia_configured())
```

Comment:
Just a suggestion: how about calling `ct_helper:get_internal_database()` isntead? Passing `mnesia` or `cets` would be easier to understand than `true` and `false`, especially that we have two boolean arguments now.